components/bl-signature-pad-component/src/action-buttons.js

```diff
@@ -0,0 +1,55 @@
+import ChangeColorButton from './buttons/change-color-button';
```

Comment:
Make export for buttons not `default`. At buttons dir create an `index.js` file and reexport this buttons:

```
export * from './change-color-button'
```

And here you may write something like that:

```
import { ClearButton,SaveJPGButton,SavePNGButton,SaveSignatureButton...} from './buttons';
```